amy/communityroles/forms.py

```diff
@@ -127,3 +127,11 @@ def clean(self) -> dict[str, Any]:
             raise ValidationError(errors)
 
         return cleaned_data
+
+    def clean_end(self):
+        '''Validate that end >= start'''
```

Comment:
Please replace `'''` with `"""`.

Alternatively you can run `black` formatter on this source file and it should get it right.